<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysSetForm</class>
 <widget class="QWidget" name="SysSetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>参数设置</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>601</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>531</width>
     <height>601</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>531</width>
     <height>601</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>频域参数</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>更改</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>段 采 样：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="secEdi">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="pixEdi">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>计算像采样：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="limEdi">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>缩 小 倍数：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="samRateEdi">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>原采样率：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>显示设置</string>
    </attribute>
    <widget class="MListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>201</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="dataSourComb">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>电流</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>电压</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>阻抗</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>线</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>数 据 源：</string>
     </property>
    </widget>
    <widget class="MListWidget" name="listWidget_1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>211</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addPb">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>190</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Src.qrc">
       <normaloff>:/new/prefix1/images/rightarrow.png</normaloff>:/new/prefix1/images/rightarrow.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="delPb">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>290</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>已选列表：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paraSavePb">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>540</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>保存参数</string>
     </property>
    </widget>
    <widget class="QLabel" name="harmLB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>谐波分量：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="harmCb">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>基波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>二次谐波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>三次谐波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>四次谐波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>五次谐波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>六次谐波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>七次谐波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>时域</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="harmLB_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>分量类型：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="RIAPCb">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>实部</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>虚部</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>相位</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>模值</string>
      </property>
     </item>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>501</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4">
       <widget class="QLineEdit" name="veiw_kEdi"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="CentralAxisLB">
        <property name="text">
         <string>底    轴：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="CentralAxisEdit"/>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>选择颜色：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="pixLB">
        <property name="text">
         <string>像 采 样：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QPushButton" name="colorTB">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="CLB">
        <property name="text">
         <string>上下位移：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="KLB">
        <property name="text">
         <string>上下缩放：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLineEdit" name="view_cEdi"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="pixSmcnpEdi"/>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <zorder>dataSourComb</zorder>
    <zorder>label_11</zorder>
    <zorder>listWidget_1</zorder>
    <zorder>addPb</zorder>
    <zorder>delPb</zorder>
    <zorder>label_12</zorder>
    <zorder>paraSavePb</zorder>
    <zorder>listWidget</zorder>
    <zorder>harmLB</zorder>
    <zorder>harmCb</zorder>
    <zorder>harmLB_2</zorder>
    <zorder>RIAPCb</zorder>
    <zorder>groupBox</zorder>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">MListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Src.qrc"/>
 </resources>
 <connections/>
</ui>
